Running SLURM prolog script on pink57
===============================================================================
Job started on Sat Aug  4 16:08:10 BST 2018
Job ID          : 123494
Job name        : saliency-array
WorkDir         : /mainfs/lyceum/grm1g17/lasagne4bio/test_saliency
Partition       : lyceum
Num hosts       : 1
Num cores       : 14
Num of tasks    : 14
Hosts allocated : pink57
Job Output Follows ...
===============================================================================
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using cuDNN version 7005 on context None
Mapped name None to device cuda: GeForce GTX 1080 Ti (0000:03:00.0)
/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
Using Theano backend.
INFO (theano.gof.compilelock): Waiting for existing lock by process '21807' (I am process '37581')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
INFO (theano.gof.compilelock): Waiting for existing lock by process '57173' (I am process '37581')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
INFO (theano.gof.compilelock): Waiting for existing lock by process '21807' (I am process '37581')
INFO (theano.gof.compilelock): To manually release the lock, delete /mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/lock_dir
Train path is downloaded ...
Loading train data ...
Loading splits ...
Using configurations: 'pureConv'
Build model
Build eval function
Load parameters
Compile functions
Traceback (most recent call last):
  File "saliency.py", line 281, in <module>
    main_saliencies_jurtz()
  File "saliency.py", line 229, in main_saliencies_jurtz
    compute_tensor_jurtz(X_train, mask_train, args)
  File "saliency.py", line 206, in compute_tensor_jurtz
    get_gradients = theano.function(inputs=[sym_x], outputs=gradients)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/compile/function.py", line 317, in function
    output_keys=output_keys)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/compile/pfunc.py", line 486, in pfunc
    output_keys=output_keys)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/compile/function_module.py", line 1841, in orig_function
    fn = m.create(defaults)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/compile/function_module.py", line 1715, in create
    input_storage=input_storage_lists, storage_map=storage_map)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/link.py", line 699, in make_thunk
    storage_map=storage_map)[:3]
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/vm.py", line 1091, in make_all
    impl=impl))
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/op.py", line 955, in make_thunk
    no_recycling)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/op.py", line 858, in make_c_thunk
    output_storage=node_output_storage)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/cc.py", line 1217, in make_thunk
    keep_lock=keep_lock)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/cc.py", line 1157, in __compile__
    keep_lock=keep_lock)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/cc.py", line 1620, in cthunk_factory
    key=key, lnk=self, keep_lock=keep_lock)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/cmodule.py", line 1166, in module_from_key
    self.refresh(cleanup=False)
  File "/lyceum/grm1g17/.conda/envs/localenv/lib/python3.5/site-packages/theano/gof/cmodule.py", line 786, in refresh
    files = os.listdir(root)
FileNotFoundError: [Errno 2] No such file or directory: '/mainfs/lyceum/grm1g17/.theano/compiledir_Linux-3.10-el7.x86_64-x86_64-with-redhat-7.4-Maipo-x86_64-3.5.5-64/tmpstzqer4e'
==============================================================================
Running epilogue script on pink57.

Submit time  : 2018-08-04T15:50:43
Start time   : 2018-08-04T16:08:10
End time     : 2018-08-04T16:09:41
Elapsed time : 00:01:31 (Timelimit=00:40:00)

Job ID: 123494
Array Job ID: 123444_44
Cluster: i5
User/Group: grm1g17/fp
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 14
CPU Utilized: 00:00:23
**** Please note that the CPU efficency of your job is less than 50% ****
**** Ask the HPC team for advice -- if you need it
**** CPU Efficiency: 1.81% of 00:21:14 core-walltime

Memory Utilized: 1.64 GB

